src/dss/config.py

```diff
@@ -9,11 +9,10 @@
 MLFLOW_DEPLOYMENT_NAME = "mlflow"
 NOTEBOOK_PVC_NAME = "notebooks"
 NOTEBOOK_IMAGES_ALIASES = {
-    "pytorch": "charmedkubeflow/jupyter-pytorch-full:1.8.0-3058193",
-    # TODO: add the rest of the rocks once updated in https://github.com/canonical/kubeflow-rocks
-    # "pytorch-cuda": "charmedkubeflow/jupyter-pytorch-cuda-full:1.8.0-xxxxx",
-    # "tensorflow-cuda": "charmedkubeflow/jupyter-tensorflow-cuda-full:1.8.0-xxxx",
-    # "tensorflow": "charmedkubeflow/jupyter-tensorflow-full:1.8.0-xxxx"
+    "pytorch": "kubeflownotebookswg/jupyter-pytorch-full:v1.8.0",
+    "pytorch-cuda": "kubeflownotebookswg/jupyter-pytorch-cuda-full:v1.8.0",
+    "tensorflow-cuda": "kubeflownotebookswg/jupyter-tensorflow-cuda-full:v1.8.0",
```

Comment:
Since we're aiming these at beginning users, should we instead say "GPU"?  It might be clearer
```suggestion
    "pytorch-gpu": "kubeflownotebookswg/jupyter-pytorch-cuda-full:v1.8.0",
    "tensorflow-gpu": "kubeflownotebookswg/jupyter-tensorflow-cuda-full:v1.8.0",
```

Reply:
nevermind, could be intel or amd gpu too